HISTORY.rst

```diff
@@ -2,6 +2,46 @@
 History
 =======
 
+v0.5.0 (2018-05-09)
```

Comment:
Date is no longer correct.

I think usually it makes sense to do a pull request for a changelog, and then, after it is merged, bump version and create a tag, using bumpversion.